src/cloudant/database.py

```diff
@@ -919,7 +919,7 @@ def delete_index(self, design_document_id, index_type, index_name):
             raise CloudantArgumentError(msg)
         index.delete()
 
-    def get_query_result(self, selector, fields=None, raw_result=False, **kwargs):
+    def get_query_result(self, selector, raw_result=False, **kwargs):
```

Comment:
I think that rather than introducing this breaking change you should return the original `get_query_result` method signature and then handle the case where `fields` is None or an empty list with `if` logic below (Line 990).